docker_kernel/kernel.py

```diff
@@ -156,25 +156,23 @@ def set_payload(self, source: str, text: str, replace: bool):
             "text": text,
             "replace": replace,
         }]
-                    
-    def tag_image(self, image_id: str, name: str, tag: str|None=None):
+
+    def tag_image(self, image: str, name: str, tag: str|None=None):
         """ Tag an image.
         Parameters
         ----------
-        image_id: str
-            Id of image to be saved.
+        image: str
+            Id or name of the image to be tagged.
         name: str
             Image name to be assigned.
         tag: str, optional
             Typically a specific version or variant of an image.
-        
+
         Return
         ------
         None
         """
-        tag = self.default_tag if tag is None else tag
 
-        if name not in self._tags:
-            self._tags[name] = {}
+        self._api.tag(image, name, tag)
+        self._kernel.send_response(f"Image {image} is tagged with: {name}:{tag if tag is not None else 'latest'}")
```

Comment:
Use `self` instead of redundant `self._kernel`

Reply:
Done